newspaper/settings.py

```diff
@@ -45,14 +43,14 @@
 MEMO_FILE = 'memoized'
 MEMO_DIR = os.path.join(TOP_DIRECTORY, MEMO_FILE)
 
-if not os.path.exists(MEMO_DIR):
-    os.mkdir(MEMO_DIR)
-
 # category and feed cache
 CF_CACHE_DIRECTORY = 'feed_category_cache'
 ANCHOR_DIRECTORY = os.path.join(TOP_DIRECTORY, CF_CACHE_DIRECTORY)
 
-if not os.path.exists(ANCHOR_DIRECTORY):
-    os.mkdir(ANCHOR_DIRECTORY)
-
 TRENDING_URL = 'http://www.google.com/trends/hottrends/atom/feed?pn=p1'
+
+for path in (TOP_DIRECTORY, MEMO_DIR, ANCHOR_DIRECTORY):
+    try:
+        os.mkdir(path)
+    except FileExistsError:
+        pass
```

Comment:
nit: I would add a comment so it's more clear of your intentions, since it's not directly obvious in this case